src/assets/style.css

```diff
@@ -44,6 +44,7 @@
 }
 
 .playkit-subtitles {
+  z-index: 10;
```

Comment:
how do subtitles look in non-audio entries after the change ? do they look ok ?

Reply:
in non-audio entries, you don't have title/description on the media itself, so it's not the same case.
however, I did encounter this issue in both types:
![image](https://github.com/kaltura/playkit-js/assets/162575174/33647d40-1b0e-4a41-ab34-1bd21d34b88c)
in this case, shall we apply z-index to all the buttons ? if so, what should be taken into account to determine the value of it?

Reply:
this is how it would look like if : 
.player .bottom-bar { z-index = 3 }
and 
.playkit-subtitles { z-index = 2 }
![image](https://github.com/kaltura/playkit-js/assets/162575174/9ebd5ba8-e213-4cba-b3dd-0ad53e8a245d)

let me know if something else needs to be considered here

Reply:
ok, i agree with what sergey said below - we should localize the change to audio entries.
theres a variable on the global state called isAudio that should help with checking the entry type. see example [here](https://github.com/kaltura/playkit-js-ui/blob/91e5b697040240a35ddf8001d494326aac0abc36/src/components/audio-entry-details/audio-entry-details.tsx#L83)